packages/frontend/i18n/src/resources/en.json

```diff
@@ -639,5 +639,48 @@
   "com.affine.auth.sign-out.confirm-modal.title": "Sign out?",
   "com.affine.auth.sign-out.confirm-modal.description": "After signing out, the Cloud Workspaces associated with this account will be removed from the current device, and signing in again will add them back.",
   "com.affine.auth.sign-out.confirm-modal.cancel": "Cancel",
-  "com.affine.auth.sign-out.confirm-modal.confirm": "Sign Out"
+  "com.affine.auth.sign-out.confirm-modal.confirm": "Sign Out",
+  "com.affine.settings.plans.recurring-yearly": "Annually",
```

Comment:
use `com.affine.payment.xxxx`, no need to emphasize settings

Reply:
This PR need to improve after https://github.com/toeverything/AFFiNE/pull/4723 merged